src/components/ReqCourse.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div
+    :class="{ 'reqcourse--min': compact, completedReqCourse: isCompletedReqCourse }"
+    class="reqcourse"
+    :style="borderColorCSSvar"
+  >
+    <div
+      v-if="!isCompletedReqCourse"
+      class="reqcourse-color"
+      :style="cssVars"
+      :class="{ 'reqcourse-color--min': compact }"
+    >
+      <img src="@/assets/images/dots/sixDots.svg" alt="dots" />
+    </div>
+    <div :class="{ 'reqcourse-content--min': compact }" class="reqcourse-content">
+      <div :class="{ 'reqcourse-main--min': compact }" class="reqcourse-main">
+        <div :class="{ 'reqcourse-top--min': compact }" class="reqcourse-top">
+          <div :class="{ 'reqcourse-code--min': compact }" class="reqcourse-code">
+            {{ courseCodeLabel }}
+          </div>
+        </div>
+        <!-- TODO: Revisit if we want this component for Incomplete Req Courses-->
+        <!-- <div v-if="!compact" class="reqcourse-name">{{ name }}</div>
+        <div class="reqcourse-info">
+          <span v-if="!compact" class="reqcourse-credits">{{ creditString }}</span>
+          <span v-if="!compact && semesterString" class="reqcourse-semesters">{{
+            semesterString
+          }}</span>
+        </div> -->
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue';
+import firebase from 'firebase/app';
+
+export default Vue.extend({
+  props: {
+    color: String,
+    subject: String,
+    number: String,
+    isCompletedReqCourse: Boolean,
+    compact: Boolean,
+  },
+  computed: {
+    borderColorCSSvar() : any {
+      return {
+        '--border-color': `#${this.color}`,
+      };
+    },
+    cssVars() : any {
+      return {
+        '--bg-color': `#${this.color}`,
+      };
+    },
+    courseCodeLabel(): string {
+      return `${this.subject} ${this.number}`;
+    },
+  },
+});
+</script>
+
+<style scoped lang="scss">
+@import '@/assets/scss/_variables.scss';
+
+.reqcourse {
+  width: 21.375rem;
+  border-radius: 0.5rem;
+  display: flex;
+  flex-direction: row;
+  background-color: $white;
+  box-shadow: -4px -4px 10px #efefef, 4px 4px 10px #efefef;
```

Comment:
This syntax is generally used for when there are two different shadows. Since it's the same color can we achieve it in the same way? Won't spend too much time on this